src/main/groovy/com/calgaryscientific/gradle/VeracodeBeginScanTask.groovy

```diff
@@ -54,41 +54,28 @@ class VeracodeBeginScanTask extends VeracodeTask {
      * @return
      */
     static Set<String> extractWhitelistModuleIds(Node xml, Set<String> whitelist) {
-        NodeList moduleList = xml.getAt("module") as NodeList
-        HashMap<String, List<String>> nonFatalModules = filterOutFatalModules(moduleList)
-        HashMap<String, List<String>> whitelistModules = getWhitelistModules(nonFatalModules, whitelist)
+        List<Node> nonFatalModules = filterOutFatalModules(XMLIO.getNodeList(xml, 'module'))
+        List<Node> whitelistModules = getWhitelistModules(nonFatalModules, whitelist)
         Set<String> missingWhitelistModules = getMissingWhitelistModules(whitelist, whitelistModules)
```

Comment:
it seems like you can extract this to something like printMissingWhiteListModules .....